### ISSUE TITLE
[Rails] API is Using Incorrect Port

### ISSUE THREAD

**Issue by styletabtech** [ga-wdi-boston/capstone-project#296]
Per @jrhorn424, I'm submitting this issue in case any other devs are accidentally using the old version of the rails-api-template like I was. The old version uses port 3000 and for the capstone it should be using port 4741. New version should be downloaded from [here](https://github.com/ga-wdi-boston/rails-api-template). 

I'll wait for further instruction on how to update my existing API to the latest version. 

Thanks!

**Comment by jrhorn424** [comment]
Thanks for reporting this!

We changed the Rails API template after we delivered it to you, but before we delivered the Ember template. So, if anyone wanted to use the copy they had already downloaded and unzipped, it wouldn't be up-to-date.

**If you haven't started working on your API**, then download a new copy of the `rails-api-template` and follow the included instructions.

**If you HAVE already started working on your API, but with an old copy of the template**, follow these steps to get it up-to-date:
- [ ] Add `rails-api-template` as a remote to your current API repository.
  - [ ] `cd my-capstone-project-api` if you aren't already in your API repository.
  - [ ] `git remote add -f rails-api-template git@github.com:ga-wdi-boston/rails-api-template.git`
- [ ] Obtain a clean working directory by making a commit or stashing any changes you have.
- [ ] Checkout the files that have recently changed. I looked at the commit history for the template and compared it to the date you started your second project. This gave me the files I needed.
  - [ ] `git checkout rails-api-template/master -- .gitignore config`
- [ ] Use `git status` and `git diff` to inspect changes. If you're happy with what you see, try it out with `bin/rails server`.
- [ ] Make a commit and note in your message that you've updated from template due to the port change.
